src/sagemaker/workflow/pipeline.py

```diff
@@ -124,6 +124,7 @@ def __init__(
         self._event_bridge_scheduler_helper = EventBridgeSchedulerHelper(
             self.sagemaker_session.boto_session.client("scheduler"),
         )
+        self.latest_pipeline_version_id = None
```

Comment:
This is just used for informational and testing purposes? Since the pipeline could be updated elsewhere which could make this inaccurate as it wouldn't be the latest version. Maybe current would be better?